bucket_56/libproxy/manifests/plist.single

```diff
@@ -0,0 +1,11 @@
+bin/proxy
+include/proxy.h
+lib/libproxy.so
+lib/libproxy.so.1
+lib/libproxy.so.1.0.0
+lib/pkgconfig/libproxy-1.0.pc
+share/cmake/Modules/Findlibproxy.cmake
+%%PERL-ON%%lib/perl5/site_perl/mach/%%PERL_VER%%/Net/Libproxy.pm
```

Comment:
replace 
lib/perl5/site_perl/mach/%%PERL_VER%% 
with
%%SITE_ARCH%%
(I believe)

Reply:
Yes this works (will be in the next commit).